server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -1126,6 +1176,18 @@ def __init__(self, term: str | Field, date_format: str, alias: str | None = None
         super().__init__("PARSE_DATE", term, date_format, alias=alias)
 
 
+class RegexpFunction(functions.Function):
+    REGEXP_LIKE = "REGEXP_LIKE"
+    REGEXP_CONTAINS = "REGEXP_CONTAINS"
+    NOT_REGEXP_LIKE = "NOT REGEXP_LIKE"
+    NOT_REGEXP_CONTAINS = "NOT REGEXP_CONTAINS"
+
+    def __init__(
+        self, keyword: str, term: str | Field, regexp_expression: str, alias: str | None = None
+    ) -> None:
+        super().__init__(keyword, term, regexp_expression, alias=alias)
```

Comment:
Shouldn't `keyword` be a RegexOp ? Otherwise, I don't think this constructor is required, since it only calls `super().__init__`

Reply:
True, the constructor is not needed !
For the name i rather prefer this class to stay as `RegexFunction` to not collide with the `RegexOp` class from `operators` what do you think ?
I updated that here : https://github.com/ToucanToco/weaverbird/pull/1418/commits/a3c4751870350c89f8da3617ccad6f4ea3dfc30d

Reply:
removed here : https://github.com/ToucanToco/weaverbird/pull/1418/commits/d28d51bfb69073501f9bb829c725240ed122c65d